check_pgactivity

```diff
@@ -1917,6 +1919,9 @@ sub check_backends_status {
     delete $status{'idle in transaction (aborted)'}
         if $hosts[0] < $PG_VERSION_90;
 
+    delete $status{'other wait event'}
+        if $hosts[0] < $PG_VERSION_96;
```

Comment:
Wow, I didn't notice. I already used the good one in another patch. Fix is on the way.